docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/OneClassMatrixFactorizationWithOptions.cs

```diff
@@ -9,60 +9,80 @@ namespace Samples.Dynamic.Trainers.Recommendation
+            // Create a matrix factorization trainer which takes "Value" as the
+            // training label, "MatrixColumnIndex" as the matrix's column index, and
+            // "MatrixRowIndex" as the matrix's row index. Here nameof(...) is used
+            // to extract field
             // names' in MatrixElement class.
             var options = new MatrixFactorizationTrainer.Options
             {
-                MatrixColumnIndexColumnName = nameof(MatrixElement.MatrixColumnIndex),
-                MatrixRowIndexColumnName = nameof(MatrixElement.MatrixRowIndex),
-                LabelColumnName = nameof(MatrixElement.Value),
-                NumberOfIterations = 20,
-                NumberOfThreads = 8,
-                ApproximationRank = 32,
-                Alpha = 1,
-                // The desired values of matrix elements not specified in the training set.
-                // If the training set doesn't tell the value at the u-th row and v-th column,
-                // its desired value would be set 0.15. In other words, this parameter determines
-                // the value of all missing matrix elements.
+                MatrixColumnIndexColumnName = nameof(
+                    MatrixElement.MatrixColumnIndex),
+                    MatrixRowIndexColumnName = nameof(MatrixElement.MatrixRowIndex),
+                    LabelColumnName = nameof(MatrixElement.Value),
+                    NumberOfIterations = 20,
+                    NumberOfThreads = 8,
+                    ApproximationRank = 32,
+                    Alpha = 1,
+
+                // The desired values of matrix elements not specified in the
+                // training set. If the training set doesn't tell the value at the
+                // u -th row and v-th column, its desired value would be set 0.15.
+                // In other words, this parameter determines the value of all
+                // missing matrix elements.
                 C = 0.15,
                 // This argument enables one-class matrix factorization.
-                LossFunction = MatrixFactorizationTrainer.LossFunctionType.SquareLossOneClass
+                LossFunction = MatrixFactorizationTrainer.LossFunctionType
+                    .SquareLossOneClass
             };
 
-            var pipeline = mlContext.Recommendation().Trainers.MatrixFactorization(options);
+            var pipeline = mlContext.Recommendation().Trainers.MatrixFactorization(
+                options);
 
             // Train a matrix factorization model.
             var model = pipeline.Fit(dataView);
 
-            // Apply the trained model to the test set. Notice that training is a partial 
-            var prediction = model.Transform(mlContext.Data.LoadFromEnumerable(testData));
+            // Apply the trained model to the test set. Notice that training is a
+            // partial 
+            var prediction = model.Transform(mlContext.Data.LoadFromEnumerable(
+                testData));
```

Comment:
📝 Incorrect wrapping (prefer to wrap the outer argument when it will fit on the same number of lines)